locals.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   cluster_name = format("atlas-%s-cluster", var.environment_name)
   namespace    = "atlassian"
+  osquery_env  = var.osquery_env
```

Comment:
Why this is necessary?

Reply:
Right! Must be leftovers from a different approach to pass envs. Removed.